src/pages/EventFeedbackAnalytics.tsx

```
@@ -0,0 +1,432 @@
+                        <div className="flex justify-between text-[10px] mb-1">
+                          <span className="text-gray-500">{cat}</span>
+                          <span className="font-bold text-purple-400">{val}</span>
+                        </div>
+                        <div className="w-full bg-gray-300 dark:bg-gray-700 rounded-full h-1.5">
+                          <div className="h-1.5 rounded-full bg-purple-500" style={{ width: `${(val / 5) * 100}%` }} />
+                        </div>
+                      </div>
+                    ))}
+                  </div>
+                  <div className="flex gap-2 mt-3">
+                    <button onClick={() => { setSelectedEvent(ev); setActiveTab('feedback'); }}
```

ESLint

[error] Replace `·onClick={()·=>·{·setSelectedEvent(ev);·setActiveTab('feedback');` with `⏎······················onClick={()·=>·{⏎························setSelectedEvent(ev);⏎························setActiveTab("feedback");⏎·····················`

(prettier/prettier)